src/scripts.js

```diff
@@ -125,4 +200,4 @@ function displayRecipePreview(current, view) {
     show(ingredientSidebar);
 
```

Comment:
We moved some the hide and show functions that were in this function when we merged last because we were also using findId() to add to the saved recipe page, but we will use what you have here. Just so you can expect to see that when we merge